test/yargs-parser.js

```diff
@@ -1218,6 +1218,21 @@ describe('yargs-parser', function () {
         result.should.have.property('someOption', 'asdf')
       })
 
+      it('should not handle 1 char aliases with different cases as the same', function () {
```

Comment:
great catch, sorry for the slow turn around on the review; this fix makes sense to me, but perhaps a better test description would be:

`should not apply camel-case logic to 1-character options`?